.vscode/launch.json

```diff
@@ -177,6 +177,15 @@
       "args": [
         "status",
       ],
+    },
+    {
+      "name": "cnquery-shell",
+      "type": "go",
+      "request": "launch",
+      "program": "${workspaceRoot}/apps/cnquery/cnquery.go",
+      "args": [
+        "shell", "ssh", "administrator@18.215.249.49", "--password", "8!dVWy&WRh)kgaTHB=7A9Chfw9v9I.94",
```

Comment:
we should remove that

Reply:
I 100% agree. Machine is destroyed, but anyway